app/assets/javascripts/components/CartedProducts.jsx

```diff
@@ -0,0 +1,65 @@
+var CartedProducts = React.createClass({
+  getInitialState: function(){
+     return {
+       initialItems: this.props.carted_products,
+       carted_products: []
+     }
+  },
+  componentWillMount: function(){
+    this.setState({carted_products: this.state.initialItems})
+  },
+  updateQuantity: function(val, id) {
+    console.log(val, "from parent && id: ", id);
+    this.props.carted_products.map((carted_product) =>
+      (carted_product.id == id) ? (
+        carted_product.quantity = val
+      ) : (
+        carted_product
+      )
+    );
+    $.ajax({
```

Comment:
@DillonStreator You're right. We already have jquery for now, and we need to make this ajax call...so we'll use it. I think main reason to get away from jquery when using react is because they are both libraries that are made to modify the DOM.